### PR TITLE
Added CodinGame

### PR DESCRIPTION
**Pull request by Vijaykv5**
## Changes proposed

Added CodinGame Website Link for Practice & Learn Programming with Interactive Challenges

### CODE REVIEW

File: database/frontend/online-code-editors.json [hunk 1]
```diff
@@ -152,6 +152,13 @@
     "url": "https://www.jdoodle.com/",
     "category": "frontend",
     "subcategory": "online-code-editors"
+  },
+  {
+    "name": "CodinGame",
+    "description": "A website to learn and practice programming with interactive challenges in different programming languages ",
```
Comment (CBID2) [inline database/frontend/online-code-editors.json:158]: [Proofreading] The description was a bit unclear. Consider using the below suggestion to help you: 
```suggestion
    "description": "This is a great website to learn and practice your coding skills. It provides interactive challenges in different programming languages ",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Welcome back @Vijaykv5! :) Your PR is off to a great start. However, the description could use some clarification, so I provided some suggestions on how to help you. :)

**Review by CBID2 — APPROVED** [review]
Well done @Vijaykv5! :) You need another approved review! :)

**Review by Anmol-Baranwal — COMMENTED** [review]
An online code editor is a web-based tool that allows users to write, edit, and run code directly in a web browser. It provides a convenient environment for developers to write and test their code without the need for installing and setting up a local development environment.

Code editors typically include features like syntax highlighting, code completion, version control integration, debugging tools, and the ability to run the code within the editor itself.

On the other hand, CodinGame is primarily a programming game platform that offers coding challenges and competitions.

@rupali-codes 
is the category correct?

**Comment by Vijaykv5** [comment]
> An online code editor is a web-based tool that allows users to write, edit, and run code directly in a web browser. It provides a convenient environment for developers to write and test their code without the need for installing and setting up a local development environment.
> 
> Code editors typically include features like syntax highlighting, code completion, version control integration, debugging tools, and the ability to run the code within the editor itself.
> 
> On the other hand, CodinGame is primarily a programming game platform that offers coding challenges and competitions.
> 
> @rupali-codes is the subcategory correct?

Yeah right?

**Comment by rupali-codes** [comment]
> An online code editor is a web-based tool that allows users to write, edit, and run code directly in a web browser. It provides a convenient environment for developers to write and test their code without the need for installing and setting up a local development environment.
> 
> Code editors typically include features like syntax highlighting, code completion, version control integration, debugging tools, and the ability to run the code within the editor itself.
> 
> On the other hand, CodinGame is primarily a programming game platform that offers coding challenges and competitions.
> 
> @rupali-codes is the subcategory correct?

i think so

**Review by rupali-codes — APPROVED** [review]

**Pull request merged**